modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -2,162 +2,160 @@
 
 = Setting up {org-name} {product-name} on OpenShift
 
-To use {product-name}, you must run OpenShift and install an add-on which enables {product-name}. For more information about OpenShift, see link:https://www.openshift.org/[OpenShift website, window="_blank"].
-
-NOTE: {org-name} recommends running OpenShift using the method described in this document. However, there are many ways to run OpenShift, and {product-name} is compatible with most of these methods. There is a known issue with Minishift that makes it unsuitable for mobile development. This issue is described in link:https://github.com/minishift/minishift/issues/1287[Issue 1287, window="_blank"].
+To use {product-name}, you must have OpenShift instance and run installation script which enables {product-name}. For more information about OpenShift, see link:https://www.openshift.org/[OpenShift website, window="_blank"].
 
 [[prerequisites]]
 == Prerequisites
 
-* MacOS or Linux
-+
-NOTE: Due to an link:https://github.com/openshift/origin/issues/18747[OpenShift bug], {product-name} will not run on RHEL/CentOS 7.4 or RHEL/CentOS 7.5.
```

Comment:
is this fixed?

Reply:
This was specific to OpenShift 3.9.0 and `oc cluster up`. Tech preview targets OpenShift 3.11. So this won't be problem.